### DISCUSSION TITLE
How to use in 5.4.3?

### DISCUSSION THREAD

**Discussion by MacLove13** [Sixze/ALS-Refactored#530] [General]
How to use in 5.4.3?

**Comment by Magic-Summer** [comment]
Until @Sixze releases the 5.4 version, I use the workaround: after you have set everything up, change the line in the `ALS.uplugin`  to `"EngineVersion": "5.4.0",` 
Notice the = (zero) at the end because we don't need the bug-fix version, which is now 3.
[Details](https://github.com/Sixze/ALS-Refactored/discussions/480)

**Reply by mustafakalaycii** [reply]
@Magic-Summer i have successfully upgraded the plugin to version 5.4 and everything is working smoothly in the editor, but I am having problems when I package the project have you solved this problem?

**Reply by Magic-Summer** [reply]
No, I haven't tried packaging yet. This may be what the many warnings are referring to. Have you tried this [workaround](https://github.com/Sixze/ALS-Refactored/discussions/480#discussioncomment-9560344)?

**Reply by mustafakalaycii** [reply]
@Magic-Summer  i didn't receive any important warnings in the method I did, just when I opened the project I had packaged, nothing came that included the plugin, but everything else was normal

**Reply by Magic-Summer** [reply]
Maybe you can find some good advice relating to packaging from [Ari](https://flassari.notion.site/UE-Tips-Best-Practices-3ff4c3297b414a66886c969ff741c5ba)

**Reply by mustafakalaycii** [reply]
@Magic-Summer thank you for helping
